### PR TITLE
Adding pictures and blog post

### PR DESCRIPTION
**Pull request by CelitaS**

### CODE REVIEW

**Comment by CelitaS** [comment]
Adding blog post 1

File: 2016-10-08-Egyptian-Gazette-Views-on-Imperialism.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+tags: -imperialism-
+image:
+  header: "Cecil Rhodes"
+  thumb: "Cecil-Rhodes.png"
+  header: "Khedivial Princess"
+  thumb: "Khedivial-Princess.png"
+
+
+Although there is a constant sense of British imperialism evident throughout the Egyptian
+Gazette, it is very interesting the attitude toward this imperialism varies widely throughout the
```
Comment (whanley) [inline 2016-10-08-Egyptian-Gazette-Views-on-Imperialism.md]: remove "it is very interesting"

File: 2016-10-08-Egyptian-Gazette-Views-on-Imperialism.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+paper. For instance, there are many cases where the Egyptian Gazette provides long blocks of
+texts or ads clearly targeted toward rich Englishmen. The paper then proceeds to provide a
+sentence-long description of an invent important to African natives. This follows with the
```
Comment (whanley) [inline 2016-10-08-Egyptian-Gazette-Views-on-Imperialism.md]: invention?

File: 2016-10-08-Egyptian-Gazette-Views-on-Imperialism.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+title: "Egyptian Gazette Views on Imperialism"
+teaser: ""
+date: 2016-10-08
+author: Celita Summa
+tags: -imperialism-
+image:
+  header: "Cecil Rhodes"
+  thumb: "Cecil-Rhodes.png"
+  header: "Khedivial Princess"
+  thumb: "Khedivial-Princess.png"
```
Comment (whanley) [inline 2016-10-08-Egyptian-Gazette-Views-on-Imperialism.md:9]: Choose just one image for the header and thumb.

**Review by whanley — CHANGES_REQUESTED** [review]
Good piece, Celita. I wonder if you could tell us a bit more about the content of the invention for natives article (an example?) and/or give a summary about the death of the princess? What society did she seem to be a part of?

**Pull request merged**